playbook-website/app/views/samples/icons.html.erb

```diff
@@ -0,0 +1,172 @@
+    <% end %>
+  <% end %>
+  <%= pb_rails("card", props: { padding: "none", header: true, margin_bottom: "sm"}) do %>
+    <%= pb_rails("card/card_body", props: { padding: "md" }) do %>
+      <%= pb_rails("body", props: { text: "Color: Blue", classname: "green_color" }) %>
+      <%= pb_rails("icon", props: { icon: image_pack_url("powergon.svg"), size: "2x", classname: "green_color" }) %>
+    <% end %>
+  <% end %>
+  <%= pb_rails("card", props: { padding: "none", header: true, margin_bottom: "sm"}) do %>
+    <%= pb_rails("card/card_body", props: { padding: "md" }) do %>
+      <%= pb_rails("body", props: { text: "Color: Blue", classname: "red_color" }) %>
+      <%= pb_rails("icon", props: { icon: image_pack_url("powergon.svg"), size: "2x", classname: "red_color" }) %>
+    <% end %>
+  <% end %>
+<% end %>
+
+
+<%= pb_rails("flex", props: { padding_left: "md"} ) do %>
+  <%= pb_rails("flex/flex_item") do %>
+    <%= pb_rails("title", props: {size: 3, text: "Animation & Transformation", padding_bottom: "sm"})%>
+  <% end %>
+<% end %>
+
+<%= pb_rails("flex", props: { padding_left: "md" }) do %>
+  <%= pb_rails("card", props: { padding: "none", header: true, margin_bottom: "sm"}) do %>
+    <%= pb_rails("card/card_body", props: { padding: "md" }) do %>
+      <%= pb_rails("icon", props: { icon: image_pack_url("nitro.svg"), size: "2x", spin: true }) %>
+    <% end %>
+  <% end %>
+  <%= pb_rails("card", props: { padding: "none", header: true, margin_bottom: "sm"}) do %>
+    <%= pb_rails("card/card_body", props: { padding: "md" }) do %>
+      <%= pb_rails("icon", props: { icon: image_pack_url("nitro.svg"), size: "2x", pulse: true }) %>
+    <% end %>
+  <% end %>
+  <%= pb_rails("card", props: { padding: "none", header: true, margin_bottom: "sm"}) do %>
+    <%= pb_rails("card/card_body", props: { padding: "md" }) do %>
+      <%= pb_rails("icon", props: { icon: image_pack_url("nitro.svg"), size: "2x" }) %>
+    <% end %>
+  <% end %>
+  <%= pb_rails("card", props: { padding: "none", header: true, margin_bottom: "sm"}) do %>
+    <%= pb_rails("card/card_body", props: { padding: "md" }) do %>
+      <%= pb_rails("icon", props: { icon: image_pack_url("nitro.svg"), size: "2x", rotation: 90 }) %>
+    <% end %>
+  <% end %>
+  <%= pb_rails("card", props: { padding: "none", header: true, margin_bottom: "sm"}) do %>
+    <%= pb_rails("card/card_body", props: { padding: "md" }) do %>
+      <%= pb_rails("icon", props: { icon: image_pack_url("nitro.svg"), size: "2x", rotation: 180 }) %>
+    <% end %>
+  <% end %>
+  <%= pb_rails("card", props: { padding: "none", header: true, margin_bottom: "sm"}) do %>
+    <%= pb_rails("card/card_body", props: { padding: "md" }) do %>
+      <%= pb_rails("icon", props: { icon: image_pack_url("nitro.svg"), size: "2x", rotation: 270 }) %>
+    <% end %>
+  <% end %>
+<% end %>
+
+<style>
+  .blue_color {
+    color: blue
+  }
+  .red_color {
+    color: red
+  }
+  .green_color {
+    color: green
+  }
+</style>
```

Comment:
```suggestion
```

Reply:
This has been all removed, wrapping the icon kit in body kit can change their color